synapseutils/sync.py

```diff
@@ -837,11 +879,8 @@ def _checkProvenace(item, path):
 
         elif not utils.is_url(item) and (utils.is_synapse_id_str(item) is None):
             raise SynapseProvenanceError(
-                (
-                    "The provenance record for file: %s is incorrect.\n"
-                    "Specifically %s, is neither a valid URL or synapseId."
-                )
-                % (path, item)
+                "The provenance record for file: %s is incorrect.\n"
```

Comment:
nit: We are moving towards using f-strings over placeholders.

https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/

Reply:
Sounds good - instead of resolving this in this PR, what do you think of me creating a low-priority Jira ticket for formatting updates in the codebase (using f-strings & making sure lines do not exceed 88 chars), and marking it as "related to" the python client refactoring initiative in 2024? @BryanFauble @thomasyu888

Reply:
@jaymedina Sounds like a great plan. We'll be chipping away at these are we revisit portions of the code base, but it'll be nice to have this as something can focus on later.

Reply:
I've made the [Jira ticket](https://sagebionetworks.jira.com/browse/SYNPY-1388) for this. Feel free to update as needed.